client/my-sites/checkout/checkout-thank-you/google-voucher/index.jsx

```diff
@@ -238,12 +238,7 @@ class GoogleVoucherDetails extends Component {
 					icon={ <img src="/calypso/images/upgrades/adwords.svg" /> }
 					title={ translate( 'Google AdWords credit' ) }
 					description={ translate(
-						'Use your {{strong}}$100{{/strong}} in credit with Google to bring the right traffic to your most important Posts and Pages.',
-						{
-							components: {
-								strong: <strong />,
-							},
-						}
+						'Use a $100 credit with Google to bring traffic to your most important Posts and Pages.'
```

Comment:
@joanrho, in the previous version $100 was emphasized with a bold font. Is the change deliberate?

Reply:
Yep, it's deliberate. Thanks so much for the detailed catch though!